charts/xui-ao-webapp/values.yaml

```diff
@@ -39,6 +49,12 @@ nodejs:
     FEATURE_SECURE_COOKIE_ENABLED: true
     FEATURE_PROXY_ENABLED: false
     FEATURE_HELMET_ENABLED: true
+    FEATURE_REDIS_ENABLED: true
+
+    # Redis
+    REDIS_KEY_PREFIX: 'activity:'
+    REDIS_TTL: 6000
+
   keyVaults:
     rpx:
       resourceGroup: rpx
```

Comment:
you will also need ao-webapp-redis-connection-string in the keyvaults secret

Reply:
Added it now